### DISCUSSION TITLE
Call failed

### DISCUSSION THREAD

**Discussion by smileart** [zed-industries/zed#17276] [Feature Requests]
So, basically we had a call with a colleague of mine, then had a break and then couldn't connect again, getting this error (see screenshot). 

Also, not sure it if could possibly be related at all, but I believe some files got inexplicably reverted to older versions, cause when I've reopened the editor it started warning me about the opened files being different from the versions on the drive (no git changes or other editors touched them though).

So all in all we couldn't continue collaborating via zed, plus had to figure out how to re-apply the reverted changes again. Really hard to trust an editor after this sort of stuff.

I wonder if anyone observed something similar? 🤔

![image](https://github.com/user-attachments/assets/151cd589-bc88-4c42-b653-d3bc526b5796)

**Comment by smileart** [comment]
It's been nearly three months as this feature stuck in the BE DB at Zed side and there's no way for me as a user to resolve this dead-lock.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support

**Reply by smileart** [reply]
@JosephTLyons Yeah, I've asked the same question there too around the same time, just to be ignored for more than a year with the issue persisting all this time. Had to abandon Zed altogether because of this, because knowing that there's no way to get help (while having BE features that I have no control over) was not an option.

**Comment by J1337** [comment]
@smileart 

Thanks for raising the issue, and sorry this didn’t get a proper response when it was originally posted. This thread is from a while back, and at the time we weren’t capable of providing active suport.

We’re putting much more of our support effort into Discord now, where the team and community are active. If you’re still running into this (or can reproduce it), could you try posting again there? We’d be happy to take another look.
